kfs-core/src/main/resources/edu/arizona/kfs/workflow/ProcurementCardDocument.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<data xmlns="ns:workflow" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="ns:workflow resource:WorkflowData">
+  <documentTypes xmlns="ns:workflow/DocumentType" xsi:schemaLocation="ns:workflow/DocumentType resource:DocumentType">
+    <documentType>
+      <name>PCDO</name>
+      <parent>FP</parent>
+      <label>Procurement Card</label>
+      <docHandler>${kfs.url}/financialProcurementCard.do?methodToCall=docHandler</docHandler>
+      <helpDefinitionURL>default.htm?turl=WordDocuments%2Fprocurementcard.htm</helpDefinitionURL>
+      <active>true</active>
+      <routingVersion>2</routingVersion>
+      <routePaths>
+        <routePath>
+          <start name="AdHoc" nextNode="AccountFullEdit" />
+          <role name="AccountFullEdit" nextNode="Account" />
+          <role name="Account" nextNode="AccountingOrganizationHierarchy" />
+          <role name="AccountingOrganizationHierarchy" nextNode="ObjectSubTypeCode" />
+          <role name="ObjectSubTypeCode" nextNode="SubFund"/>
+          <role name="SubFund" nextNode="RequiresAutoApprovalNotification" />
+          <split name="RequiresAutoApprovalNotification">
+            <branch name="True">
+              <role name="AutoApprove" nextNode="joinOnAutoApproval" />
+            </branch>
+            <branch name="False">
+              <simple name="NoOpAutoApproval" nextNode="joinOnAutoApproval" />
+            </branch>
+            <join name="joinOnAutoApproval" />
+          </split>
+        </routePath>
+      </routePaths>
+      <routeNodes>
+        <start name="AdHoc" />
+        <role name="AccountFullEdit">
+          <qualifierResolverClass>org.kuali.rice.krad.workflow.attribute.DataDictionaryQualifierResolver</qualifierResolverClass>
+        </role>
+        <role name="Account">
+          <qualifierResolverClass>org.kuali.rice.krad.workflow.attribute.DataDictionaryQualifierResolver</qualifierResolverClass>
+          <activationType>P</activationType>
+        </role>
+        <role name="AccountingOrganizationHierarchy">
+          <qualifierResolverClass>org.kuali.rice.krad.workflow.attribute.DataDictionaryQualifierResolver</qualifierResolverClass>
+          <activationType>P</activationType>
+        </role>
+        <role name="ObjectSubTypeCode">
+          <qualifierResolverClass>org.kuali.rice.kns.workflow.attribute.DataDictionaryQualifierResolver</qualifierResolverClass>
```

Comment:
should be:
org.kuali.rice.krad.workflow.attribute.DataDictionaryQualifierResolver